### ISSUE TITLE
'python build.py --include canvas' does not work!

### ISSUE THREAD

**Issue by imotai** [mrdoob/three.js#5922]
Fail to build canvas renderer
```
 * Building ../../build/three.js
Traceback (most recent call last):
  File "build.py", line 109, in <module>
    main()
  File "build.py", line 61, in main
    with open(filename, 'r', encoding='utf-8') as f:
IOError: [Errno 2] No such file or directory: '../../src/materials/SpriteCanvasMaterial.js'
```
where are the following files?
```
       "src/renderers/CanvasRenderer.js",
	"src/renderers/renderables/RenderableVertex.js",
	"src/renderers/renderables/RenderableFace.js",
	"src/renderers/renderables/RenderableObject.js",
	"src/renderers/renderables/RenderableSprite.js",
	"src/renderers/renderables/RenderableLine.js"
```

**Comment by mrdoob** [comment]
Woops!

**Comment by dubejf** [comment]
Fixed by #6513.
